cosmos/operators/azure_container_instance.py

```diff
@@ -0,0 +1,185 @@
+from __future__ import annotations
+
+from typing import Any, Callable, Sequence
+
+import yaml
+from airflow.utils.context import Context
+from cosmos.config import ProfileConfig
+
+from cosmos.log import get_logger
+from cosmos.operators.base import DbtBaseOperator
+
+logger = get_logger(__name__)
+
+# ACI is an optional dependency, so we need to check if it's installed
+try:
+    from airflow.providers.microsoft.azure.operators.container_instances import AzureContainerInstancesOperator
+except ImportError:
+    raise ImportError(
+        "Could not import AzureContainerInstancesOperator. Ensure you've installed the Microsoft Azure provider "
+        "separately or with `pip install astronomer-cosmos[...,azure-container-instance]`."
+    )
+
+
+class DbtAzureContainerInstanceBaseOperator(AzureContainerInstancesOperator, DbtBaseOperator):  # type: ignore
+    """
+    Executes a dbt core cli command in a Docker container.
+    """
```

Comment:
Can you update the docstring here
```suggestion
    """
    Executes a dbt core cli command in an Azure container instance.
    """
```

Reply:
Done!